### DISCUSSION TITLE
Open Weather API Key or Version Incorrect

### DISCUSSION THREAD

**Discussion by ccigas** [jeroenterheerdt/HAsmartirrigation#349] [General]
Hello! Opening a discussion since I dont think this is a bug. I am trying to set up Smart Irrigation for the first time ever. I made an account on Open Weather Map and got the api key under API Keys in my account. When I plug it in on set up of Smart Irrigation, I get the error, Open Weather API Key or Version Incorrect. I tried the key on both v3 and v2.5.

Is there a different key I am supposed to set up or is something else going on? 

Thank you

**Comment by jeroenterheerdt** [comment]
https://github.com/jeroenterheerdt/HAsmartirrigation?tab=readme-ov-file#getting-open-weather-map-api-key

**Reply by ccigas** [reply]
Thanks. Didn't realize I actually had to subscribe, just thought it was auto subscribed for the free version.
